### DISCUSSION TITLE
osu!taiko Difficulty Sorting Incorrect Until Recalculation

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#18450] [Q&A] [answered]
Just a question, not a request. Since the stamina rework for taiko was merged in version 2022.528.0, the song select screen no longer sorts by difficulty correctly for this ruleset. Tooltips on the beatmapset and set ordering are based on osu-web's SR rather than the new changes. Will this be adjusted or will we have to wait until the full deployment of the new SR system?
![osu_2022-05-27_20-54-40](https://user-images.githubusercontent.com/74262407/170808863-75450d5d-00e8-40f5-983f-26cc7472170e.png)

**Comment by peppy** [comment, marked as answer]
There's about 50 issues/discussions covering this. Support for reprocessing beatmaps is not implemented yet.
